### PR TITLE
Update es_es.json

### PR DESCRIPTION
**Pull request by albertosaurio65**

### CODE REVIEW

File: src/main/resources/assets/origins/lang/es_es.json [hunk 1]
```diff
@@ -1,170 +1,243 @@
...
+  "origins.gui.impact.none": "Ninguno",
+  "origins.gui.impact.low": "Bajo",
+  "origins.gui.impact.medium": "Intermedio",
+  "origins.gui.impact.high": "Alto",
+  "origins.gui.version_mismatch": "Este servidor está usando una versión de Origins (v%1$s) que es incompatible con la que tienes instalada (v%2$s)",
+  
+  "origins.gui.badge.active": "Habilidad activa. Presione %s para activarla.",
+  "origins.gui.badge.toggle": "Alternar con %s.",
+  "origins.gui.badge.recipe.crafting.shaped": "Receta de elaboración (con forma) proporcionada por el poder:",
+  "origins.gui.badge.recipe.crafting.shapeless": "Receta de elaboración (sin forma) proporcionada por el poder:"
```
Comment (EdwinMindcraft) [inline src/main/resources/assets/origins/lang/es_es.json]: You missed a comma here.
```suggestion
  "origins.gui.badge.recipe.crafting.shapeless": "Receta de elaboración (sin forma) proporcionada por el poder:",
```

File: src/main/resources/assets/origins/lang/es_es.json [hunk 2]
```diff
@@ -1,170 +1,243 @@
...
+  
+  "power.origins.velvet_paws.name": "Patas Acolchadas",
+  "power.origins.velvet_paws.description": "Las almohadillas en tus patas hacen que tus pisadas no causen vibraciones, lo que hace que no te puedan detectar los Sensores de Sculk.",
+  
+  "category.origins": "Orígenes",
+  "key.origins.primary_active": "Poder Activo (Principal)",
+  "key.origins.secondary_active": "Poder Activo (Secundario)",
+  "key.origins.view_origin": "Ver Origen",
+  
+  "text.autoconfig.origins.title":   "origins Configuration",
```
Comment (EdwinMindcraft) [inline src/main/resources/assets/origins/lang/es_es.json]: Please fix the case here.
```suggestion
  "text.autoconfig.origins.title":   "Origins Configuration",
```

**Review by EdwinMindcraft — CHANGES_REQUESTED** [review]
Two changes that I think were mistakes on your part, other than that, looks great.

**Comment by albertosaurio65** [comment]
Ok, the error of the comma was that I moved it from the bottom and I forgot to put it, the capital letter was because I forgot to update that part.

I have a doubt with two things that I don't know very well where and how it is used since it has several interpretations

`  "text.autoconfig.origins.option.xOffset": "Cooldown Bar Offset X",
  "text.autoconfig.origins.option.yOffset": "Cooldown Bar Offset Y",`

**Comment by EdwinMindcraft** [comment]
It appeared in the origins configuration screen when configured was used. Currently it shouldn't be used. The actual text is in Apoli.
In this case offset is a displacement vector, ie it would move the power bars by the given coordinates from their original position.

**Pull request merged**